### DISCUSSION TITLE
Make the announcements optional

### DISCUSSION THREAD

**Discussion by FrDH** [craftcms/cms#13728] [Ideas] [answered]
Is it possible to hide the announcements dropdown for non-admin users? IMHO the messages don't really add to the user experience of anyone other than an admin.

**Comment by piotrpog** [comment]
In the case of unpayed plugins on the live server, part of usefulness of these announcements is to create social pressure for owner of website to make purchase, the more control panel users see this message the better :)

**Comment by FrDH** [comment]
@piotrpog  I'm all for the unpayed plugins warnings (although it is kind of annoying the license doesn't transfer to the staging server) , but here I'm referring to the dropdown below the giftbox icon (see screenshot). 

![Scherm­afbeelding 2023-09-25 om 08 54 35](https://github.com/craftcms/cms/assets/1387163/4194aaec-2d90-42d7-b87a-2040c8e6a6ae)

**Comment by brandonkelly** [comment, marked as answer]
Announcements were intended to be used for features that are relevant to authors, not only developers.

I just added a new `$adminsOnly` argument to `craft\services\Announcements::push()` for the next release, and will suggest to Solspace that they start passing `true` to it. (395e625f71335e54bfbebcd5454e40f4f32c65c7)

**Reply by brandonkelly** [reply]
Craft 4.5.6 is out with that new argument.
